### PR TITLE
Support for RTL

### PR DESCRIPTION
**Pull request by yanivshaked**
This PR add functionality to support RTL text (Hebrew, Arabic, etc.)
Handles issue #50 .

### CODE REVIEW

**Comment by yanivshaked** [comment]
@marcelgarus Can you please review?

File: example/lib/main.dart [hunk 1]
```diff
@@ -17,19 +25,35 @@ class MyApp extends StatelessWidget {
             _buildComplexMarquee(),
           ].map(_wrapWithStuff).toList(),
         ),
+        floatingActionButton: FloatingActionButton.extended(
+          onPressed: () {
+            setState(() {
+              _useRtlText = !_useRtlText;
+            });
+          },
```
Comment (marcelgarus) [inline example/lib/main.dart:33]:
```suggestion
          onPressed: () => setState(() => _useRtlText = !_useRtlText),
```

File: example/lib/main.dart [hunk 2]
```diff
@@ -1,9 +1,17 @@
+import 'package:flutter/cupertino.dart';
```
Comment (marcelgarus) [inline example/lib/main.dart:1]: Do we need this?
```suggestion
```

File: example/lib/main.dart [hunk 3]
```diff
@@ -17,19 +25,35 @@ class MyApp extends StatelessWidget {
             _buildComplexMarquee(),
           ].map(_wrapWithStuff).toList(),
         ),
+        floatingActionButton: FloatingActionButton.extended(
+          onPressed: () {
+            setState(() {
+              _useRtlText = !_useRtlText;
+            });
+          },
+          label: !_useRtlText ? const Text('Switch') : const Text('החלף'),
```
Comment (marcelgarus) [inline example/lib/main.dart:34]: I'd label them "Switch to Hebrew" and the Hebrew equivalent of "Switch to English"

**Review by marcelgarus — CHANGES_REQUESTED** [review]
Looks nice, thank you!
And looks like this feature was surprisingly simple to implement 😀

File: example/lib/main.dart (hunk 3 above)
Comment (yanivshaked) [reply example/lib/main.dart]: Fixed

**Comment by yanivshaked** [comment]
All changes done according to code review.
Indeed, turned out it was very easy to implement 👍

**Comment by yanivshaked** [comment]
@marcelgarus Can you please review and release a new version?

**Comment by marcelgarus** [comment]
Looks good 👍🏻 I published this as 2.2.0

**Pull request merged**